test/test_nn.py

```diff
@@ -14198,6 +14198,14 @@ def _make_noncontiguous(inp):
             bias.requires_grad_(False)
         self.assertTrue(gradgradcheck(convolution, inputs, nondet_tol=gradcheck_nondet_tol))
 
+
+    def test_channel_shuffle_runtime_errors(self, device):
+        # gh-76616: nn.ChannelShuffle will crash with empty input tensor
+        groups = 3
+        input_tensor = torch.rand([0, 9, 4, 4], device=device)
+        with self.assertRaisesRegex(RuntimeError, "nn.ChannelShuffle: Input tensor must not be empty"):
```

Comment:
With the change to return `self`, I think this test should be tweaked to longer expect an error